app/public/dist/client/locales/en/translation.json

```diff
@@ -1256,383 +1256,383 @@
-    "DREAM_EATER": "If an enemy is SLEEP, jump next to it, deals [45,90,SP] SPECIAL and restores the same amount of HP. Otherwise, puts the target SLEEP for [2.5,5] seconds",
-    "SPARK": "The Pokémon fires a spark charged with electricity at the target, inflicting [40,80,SP] SPECIAL. The spark can bounce up to 4 times between adjacent enemies. Each rebound halves the SPECIAL",
-    "CRUNCH": "Deal [40,80,150,SP] SPECIAL. If it kills the target, heal for 50% of target's max HP",
-    "CROSS_POISON": "Deal [30,60,120,SP] SPECIAL to target and enemies adjacent to it, and POISONNED them for 3 seconds",
+    "AURASPHERE": "Exude an aura and project energy in a line, dealing [25,50,SP] SPECIAL and inflicting all enemy Pokémon hit with SILENCE for 3 seconds.",
+    "SKETCH": "Copy the ability, max PP and STAR of the allied Pokémon on the user's left at the start of the fight.",
+    "OVERDRIVE": "Twang an electric guitar, causing a huge echo and strong vibration. Deal [120,SP]% of ATK as SPECIAL within a 3 tile radius of the user, -20% per tile from the user.",
+    "LOVELY_KISS": "Kiss the enemy Pokémon with a frightening expression. Put the target to SLEEP for [2,4,6,SP] seconds. If the target is already suffering from SLEEP, deal [50,100,150,SP] SPECIAL instead.",
+    "TRANSFORM": "Take the appearance and the stats of the target (except HP).",
+    "PSYCHIC_FANGS": "Deal [100,SP] PHYSICAL and remove all the target's ATK, DEF and SPE_DEF stat increases.",
+    "SHED_TAIL": "Create a substitute and charge towards the ally with the lowest HP, granting them [80, SP] SHIELD.",
+    "NIGHT_SLASH": "Deal [15,30,60,SP] SPECIAL to the target. Can crit by default.",
+    "KOWTOW_CLEAVE": "Deal 150% ATK to the target, + [20,SP]% TRUE for each fallen allied Pokémon. Can crit by default",
+    "SHIELDS_DOWN": "Transform Minior to a random core form.",
+    "SHIELDS_UP": "Transform Minior to its meteor form.",
+    "SANDSEAR_STORM": "Searingly hot sand that strikes all enemy Pokémon in a straight line, dealing [80,SP] SPECIAL and applying BURN for 2 seconds.",
+    "WILDBOLT_STORM": "Thunderous tempest that strikes all enemy Pokémon in a straight line, dealing [80,SP] SPECIAL and inflicting PARALYSIS for 4 seconds.",
+    "BLEAKWIND_STORM": "Savagely cold winds that strike all enemy Pokémon in a straight line, dealing [80,SP] SPECIAL and applying FREEZE for 2 seconds.",
+    "SPRINGTIDE_STORM": "Fierce winds brimming with love and hate strike all enemy Pokémon in a straight line, dealing [80,SP] SPECIAL and inflicting CHARM for 2 seconds.",
+    "AURA_WHEEL": "Deal [60,SP] SPECIAL to the target, gain [10,SP]% ATK_SPEED and switch between Normal and Hangry form.",
+    "LICK": "Deal [30,60,120,SP] SPECIAL to the target and inflict them with CONFUSION and PARALYSIS for 3 seconds.",
+    "FURY_SWIPES": "Deal 100% of ATK as PHYSICAL to the target [2,SP] to [5,SP] times.",
```

Comment:
not sure what the 100% of ATK brings to the table compared to original

Reply:
The original was a bit awkward and unclear. I feel like there's a missing BASIC_ATK keyword or something that would fit better here. "Deal 100% of ATK as PHYSICAL" is consistent with other abilities that use a similar wording.

Reply:
Yeah that's much better. Fixed